gapic/src/main/com/google/gapid/views/FramebufferView.java

```diff
@@ -128,14 +139,37 @@ public FramebufferView(Composite parent, Models models, Widgets widgets) {
 
     setLayout(new GridLayout(2, false));
 
-    attachmentListener = new AttachmentListener(widgets.theme);
+    Composite header = createComposite(this, new GridLayout(2, false));
 
-    toolBar = createToolBar(widgets.theme);
+    picker = new AttachmentPicker(this);
```

Comment:
Can we create the picker at the right side of the toolbar? So that we can avoid the movement and blankness of the toolbar after clicking show attachment.

Reply:
We are basing this on UI mockups linked in go/gbuffer-strip. Direct link: https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZUyss3W9BIEB/files/MCEJu8Y2hyDScXfsMX7dIYFTKmMLUPLxbBY

We went with the top one because it was more discoverable, which was a problem with the original we wanted to address. In the bottom is where it is part of the toolbar like before.

However, you are suggesting to have it function like the top (not part of the toolbar, has a show/hide button), but look like the bottom in that it is to the side of the toolbar, correct? I can see the appeal to that. 

Anyone with second opinions?